src/event.rs

```diff
@@ -143,6 +144,17 @@ pub enum Event<'a, T: 'static> {
   /// - **Other**: Unsupported.
   #[non_exhaustive]
   Reopen { has_visible_windows: bool },
+
+  /// ## Push Tokens
+  ///
+  /// Emitted when registration completes and an application push token is made available; on Apple
+  /// platforms, this is the APNS token. On Android, this is the FCM token.
+  PushRegistration(PushToken),
+
+  /// ## Push Token Errors
+  ///
+  /// Emitted when push token registration fails.
+  PushRegistrationError(String),
```

Comment:
needs guard

Reply:
not sure what you mean by "needs guard"

Reply:
@amrbashir these are my own comments to remind myself to add `#[cfg(feature = "push-notifications") ` so that this code isn't included unless developers opt-in.